docs/design/features/assembly-conflict-resolution.md

```diff
@@ -17,19 +17,12 @@ Since each layer (and other probing locations) can contain the same assembly by
 #### Probe Ordering
 The probing locations consists of:
 1.  Serving Location
-1.  Shared Store
-1.  Framework directory(s) from higher to lower
 1.  App directory
-1.  Additional locations specified in runtimeconfig.dev.json file
-
-For example, here's the probing order on Windows when running a non-published folder (in order to get additional locations from the app.runtimeconfig.dev.json file)
-* `C:\\Program Files (x86)\\coreservicing\\x64`
-*	`C:\\Program Files (x86)\\coreservicing\\pkgs`
-*	`C:\\Program files\\dotnet\\shared\\Microsoft.NETCore.App\\2.0.0`
-*	(app location)
-*	`C:\\Users\\<user>\\.nuget\\packages`
-	`C:\\Program Files (x86)\\Microsoft SDKs\\NuGetPackagesFallback`
-*	`C:\\Program Files\\dotnet\\sdk\\NuGetFallbackFolder`
+1.  Framework directory(s) from higher to lower
```

Comment:
The doc is no longer self-consistent unforetunately
In the section marked 'Algorithm' the text refers to app being after framework.
In the section marked 'Changes for 2.1+' it is refers back to this list with the expectation that framework is item 3 and app is item 4.

Reply:
Good catch. I rewrote the doc almost fully to only describe the current behavior - it's not very useful to have a "proposal" doc around since some pieces of it were never implemented and it's hard to read to determine the current behavior.